extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/supervisor/KafkaSupervisorTuningConfigTest.java

```diff
@@ -24,7 +24,6 @@
 import org.apache.druid.indexing.kafka.KafkaIndexTaskModule;
 import org.apache.druid.jackson.DefaultObjectMapper;
 import org.apache.druid.segment.IndexSpec;
-import org.apache.druid.segment.data.CompressionStrategy;
```

Comment:
@SandishKumarHN please self-review unrelated changes via Github interface from `extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/supervisor/KafkaSupervisorTuningConfigTest.java` onwards.

Reply:
@SandishKumarHN did you make these changes yourself, or they result from an unclear merge with master?

Reply:
@leventov I merged with the local master branch instead of the upstream/master!

Reply:
@SandishKumarHN please reconcile with the upstream master and get your branch to the state when there are no changes which appear as unrelated to the purpose of this PR (for example the one to which this comment thread is attached).